### PR TITLE
Add -r/--requirements and -e/--exclude options.

### PR DESCRIPTION
**Pull request by prologic**
This adds two capabilities to this lovely little tool (_Thank you_)
- The ability to use and specify a `requirements.txt` file.
- The ability to exclude some built wheels from the build directory before syncing to S3 Bucket (_see below_).

The use-case for the `-e/--exclude` option is to avoid installing `Django==1.4.19` as a wheel because of this [bug](https://code.djangoproject.com/ticket/19252) as talked about in this [post](http://blog.ionelmc.ro/2013/08/26/django-pip-wheel/)

### CODE REVIEW

File: mkwheelhouse.py [hunk 1]
```diff
@@ -93,7 +94,11 @@ def build_wheels(packages, index_url):
         '--wheel-dir', temp_dir,
         '--find-links', index_url
     ]
-    args += packages
+    # Use a requirements.txt file -r/--requirements
+    if requirements is not None:
+        args += ['-r', requirements]
+    else:
+        args += packages
```
Comment (benesch) [inline mkwheelhouse.py]: This seems surprising to me: specifying a requirements file will cause any additionally specified packages to be silently ignored. We should either warn the user that they can't specify both `args.requirement` and `args.package` or always append any directly-specified packages.

**Comment by benesch** [comment]
Thanks for the beautifully-formatted patch! Glad you find the tool helpful.

We actually _had_ support for installing from requirements files, but I [removed the feature](https://github.com/WhoopInc/mkwheelhouse/commit/1a4d21ba3881b948c2279f2b67d4706bcbb638eb) because we converted all our internal applications to include a `setup.py` dependency list. But you needing the feature is reason enough to reinclude it! Sorry that you had to duplicate the work there.

If you could address the one comment I made, this is ready to go! Otherwise I'll try to get to it soon.

File: mkwheelhouse.py (hunk 1 above)
Comment (prologic) [inline mkwheelhouse.py]: Agreed I'll update the patch

**Comment by prologic** [comment]
Done :)

**Comment by benesch** [comment]
Committed in 27fe0e72a7. Thanks!

**Comment by prologic** [comment]
Most welcome!

**Pull request closed without merging**